docs/images/Bluesky_Logo.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="600" height="530" version="1.1" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Too big! 🤣 
See @melissawm link to the preview.

Reply:
It should be fixed. Let's wait on preview.